src/locales/ar.po

```diff
@@ -720,6 +733,11 @@ msgstr "رجاء أدخل عنوان بريدك الإلكتروني."
 msgid "Please log in."
 msgstr "الرجاء تسجيل الدخول."
 
+#, fuzzy
+#| msgid "Please note that your settings have been updated as follows:"
+msgid "Please note that client %1 settings has been updated as below:"
+msgstr "يرجى ملاحظة أن الإعدادات الخاصة بك تم تحديثها كما يلي:"
```

Comment:
this fuzzy text will happen whenever we change something in sentence, this is way to tell translators to fix this as something has changed.